assets/app/view/game/part/city_slot.rb

```diff
@@ -82,7 +82,10 @@ def on_click(event)
           entity = @selected_company || step.current_entity
           actions = step.actions(entity)
           return if (%w[remove_token place_token] & actions).empty?
-          return if @token && !step.can_replace_token?(entity, @token)
+          return if @token && !step.can_replace_token?(entity, @token) &&
```

Comment:
can we use block syntex 

entity.abilities(:token) do |token|
 ...

for the future of multiple abitiies